authorization/invitation/repository/invitation.go

```diff
@@ -192,6 +192,26 @@ func (m *GormInvitationRepository) Delete(ctx context.Context, id uuid.UUID) err
 
 	obj := Invitation{InvitationID: id}
 
+	var invitationRoles []InvitationRole
+
+	err := m.db.Where("invitation_id = ?", id).Find(&invitationRoles).Error
+	if err != nil && err != gorm.ErrRecordNotFound {
```

Comment:
You have a test missing for `Invitation not found`.

Reply:
Test added.